### PR TITLE
startPoolRange and endPoolRange Name changed

### PR DESCRIPTION
**Pull request by alonSadan**
changed startPoolRange to startRange and endPoolRange to endRange,
since it is obvious that it is the pool range.

### CODE REVIEW

**Comment by ovirt-infra** [comment]
Hello contributor, thanks for submitting a PR for this project! 

I am the bot who triggers "standard-CI" builds for this project.
As a security measure, I will not run automated tests on PRs that are not from white-listed contributors.

In order to allow automated tests to run, please ask one of the project maintainers to review the code and then do one of the following:

1. Type `ci test please` on this PR to trigger automated tests for it.
2. Type `ci add to whitelist` on this PR to trigger automated tests for it and also add you to the contributor white-list so that your future PRs will be tested automatically. ( keep in mind this list might be overwritten if the job XML is refreshed, for permanent whitelisting, please follow #3 option )
3. If you are planning to contribute to more than one project, maybe it's better to ask them to add you to the project organization, so you'll be able to run tests for all the organization's projects.

File: README.md [hunk 1]
```diff
@@ -128,8 +128,8 @@ metadata:
   name: cluster
 spec:
   kubeMacPool:
-   startPoolRange: "02:00:00:00:00:00"
-   endPoolRange: "FD:FF:FF:FF:FF:FF"
+   startRange: "02:00:00:00:00:00"
```
Comment (phoracek) [inline README.md]: Maybe I misunderstood Dan's comment, but I thought the request was to change it to "rangeStart"/"rangeStop". @dankenigsberg?

Comment (phoracek) [reply README.md]: > I'm not a native English speaker, but I think 'StartPoolRange' is not proper. I'd prefer to call it 'RangeStart'; it is quite clear that it is an attribute of the Pool.

Comment (alonSadan) [reply README.md]: you are right. my mistake

File: data/kubemacpool/002-configmap.yaml [hunk 2]
```diff
@@ -1,8 +1,8 @@
 ---
 apiVersion: v1
 data:
-  START_POOL_RANGE: {{ .StartPoolRange }}
-  END_POOL_RANGE: {{ .EndPoolRange }}
+  START_POOL_RANGE: {{ .StartRange }}
```
Comment (SchSeba) [inline data/kubemacpool/002-configmap.yaml]: I think we should also update the https://github.com/K8sNetworkPlumbingWG/kubemacpool and change it to `RANGE_START` 

@phoracek @dankenigsberg  what you think?

Comment (phoracek) [reply data/kubemacpool/002-configmap.yaml]: I woudn't mind. There won't be a better time to do that than now.

Comment (dankenigsberg) [reply data/kubemacpool/002-configmap.yaml]: ack for `RANGE_START`. We should try to use proper English, and should avoid the object name if it is obvious from context.

File: README.md (hunk 1 above)
Comment (dankenigsberg) [reply README.md]: I think that rangeStart/rangeEnd is better English.

**Comment by SchSeba** [comment]
@alonSadan please update the kubemacpool image to use `quay.io/kubevirt/kubemacpool:v0.2.0`

File: pkg/components/components.go [hunk 3]
```diff
@@ -18,7 +18,7 @@ const (
 	LinuxBridgeCniImageDefault = "quay.io/kubevirt/cni-default-plugins:latest"
 	SriovDpImageDefault        = "quay.io/booxter/sriov-device-plugin:latest"
 	SriovCniImageDefault       = "docker.io/nfvpe/sriov-cni:latest"
-	KubeMacPoolImageDefault    = "quay.io/schseba/mac-controller:latest"
+	KubeMacPoolImageDefault    = "quay.io/kubevirt/kubemacpool:latest"
```
Comment (phoracek) [inline pkg/components/components.go]: Please use tagged version. What should be the tag @SchSeba ?

File: pkg/network/kubemacpool.go [hunk 4]
```diff
@@ -23,21 +23,21 @@ func validateKubeMacPool(conf *opv1alpha1.NetworkAddonsConfigSpec) []error {
...
-	if _, err := net.ParseMAC(conf.KubeMacPool.StartPoolRange); err != nil {
-		return []error{errors.Errorf("failed to parse startPoolRange invalid mac address")}
+	if _, err := net.ParseMAC(conf.KubeMacPool.RangeStart); err != nil {
+		return []error{errors.Errorf("failed to parse RangeStart because the mac address is invalid")}
 	}
 
-	if _, err := net.ParseMAC(conf.KubeMacPool.EndPoolRange); err != nil {
-		return []error{errors.Errorf("failed to parse endPoolRange invalid mac address")}
+	if _, err := net.ParseMAC(conf.KubeMacPool.RangeEnd); err != nil {
+		return []error{errors.Errorf("failed to parse RangeEnd because the mac address is invalid")}
```
Comment (SchSeba) [inline pkg/network/kubemacpool.go]: RangeEnd -> rangeEnd

File: pkg/components/components.go (hunk 3 above)
Comment (phoracek) [reply pkg/components/components.go]: Actually, maybe it deserves its own PR. Nevermind

File: pkg/apis/networkaddonsoperator/v1alpha1/networkaddonsconfig_types.go [hunk 5]
```diff
@@ -22,8 +22,8 @@ type Multus struct{}
 type LinuxBridge struct{}
 
 type KubeMacPool struct {
-	StartPoolRange string `json:"startPoolRange,omitempty"`
-	EndPoolRange   string `json:"endPoolRange,omitempty"`
+	RangeStart string `json:"RangeStart,omitempty"`
```
Comment (SchSeba) [inline pkg/apis/networkaddonsoperator/v1alpha1/networkaddonsconfig_types.go]: RangeStart -> rangeStart

File: pkg/apis/networkaddonsoperator/v1alpha1/networkaddonsconfig_types.go [hunk 6]
```diff
@@ -22,8 +22,8 @@ type Multus struct{}
...
+	RangeEnd   string `json:"RangeEnd,omitempty"`
```
Comment (SchSeba) [inline pkg/apis/networkaddonsoperator/v1alpha1/networkaddonsconfig_types.go]: RangeEnd -> rangeEnd

File: pkg/apis/networkaddonsoperator/v1alpha1/networkaddonsconfig_types.go (hunk 5 above)
Comment (phoracek) [reply pkg/apis/networkaddonsoperator/v1alpha1/networkaddonsconfig_types.go]: Only the one on right. Left-hand-side RangeStart should remain this way

**Pull request merged**